common/services/move.js

```diff
@@ -42,22 +42,35 @@ const moveService = {
       })
   },
 
-  getMovesByDateRangeAndStatus({
-    dateRange = [],
-    status,
-    fromLocationId,
-  } = {}) {
+  getMovesByDateRangeAndStatus({ dateRange = [], status, locationId } = {}) {
```

Comment:
Is there a reason this was updated? Feels like the name should still be `fromLocationId` as that is what the service method expects and is using this value for?

Reply:
I think the current name is clearer. We get the moves by range and status. Location may not be passed if it's not needed.

Reply:
Sorry, I meant the name of the variable that you renamed. You renamed it from `fromLocationId` to `locationId` - but as far as I can tell the functionality is still the same, ie it is actually the `fromLocationId`, not just an arbitrary `locationId`. It is then specifically passed into the `from_location` filter. So that's why I thought it would make more sense to leave this unchanged.

Reply:
Ahhh ... ! Ok. I see what you mean.